services/docker-compose.yml

```diff
@@ -390,6 +390,12 @@ services:
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED}
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_INTERVAL_SEC=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_INTERVAL_SEC}
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_COUNTER_FAIL=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_COUNTER_FAIL}
+      - RESOURCE_USAGE_TRACKER_S3_ENABLED=${RESOURCE_USAGE_TRACKER_S3_ENABLED}
+      - S3_ACCESS_KEY=${S3_ACCESS_KEY}
+      - S3_BUCKET_NAME=${S3_EXPORT_CSV_BUCKET_NAME}
+      - S3_ENDPOINT=${S3_ENDPOINT}
+      - S3_SECRET_KEY=${S3_SECRET_KEY}
+      - S3_SECURE=${S3_SECURE}
```

Comment:
Question: how sustainable is this?
so you use the very same S3 access/secret as the standard S3 access but use a different bucket. this looks like something that could very easily go wrong.
Also none of them are in .env-devel (is the policy where the .env-devel in here has to have the exact same ENVs as in osparc-ops-config gone? If yes why and what is now the new rationale? And then why am I the only one still abiding to it?)

Reply:
As discussed I will create a new set of S3 credentials just for this usecase